src/main/java/net/tardis/mod/capability/CapabilityTardis.java

```diff
@@ -309,8 +309,8 @@ public void update() {
 						}
 					}
 					
-					
-					if (player.world.getBlockState(player.getPosition().down()).getBlock() != Blocks.AIR) {
+					Block blockToCheck = player.world.getBlockState(player.getPosition().down()).getBlock();
+					if (blockToCheck != Blocks.AIR && blockToCheck != Blocks.WATER) {
```

Comment:
Check against the blockstates material instead, I think you can just use isLiquid 

For air, in 1.13+ there will be different types of air, so you should use player.world.isAirBlock(player.getPosition().down()) for the air check

Reply:
Done, I don't see any similar method for water, so I think all is ok now

Reply:
I think there is a method for ALL liquids in iBlockState?
Removing getBlock will return a IBlockState instead of a block, from that, you should be able to use isLiquid?

Reply:
I don't think so
![image](https://user-images.githubusercontent.com/16562647/55582055-c5c26e80-571e-11e9-97bb-b1316c936bf7.png)

I don't think the solution is in these options because the leaves can be a problem

Reply:
block.getMaterial().isLiquid() ?

Reply:
Deprecated
![image](https://user-images.githubusercontent.com/16562647/55582328-7cbeea00-571f-11e9-87a0-69c31a9b4d9f.png)

Reply:
I'll use IBlockState

Reply:
Ok, getMaterial is under IBlockState